bin/check-ts.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+# Script to run the type-checker in all directories in the npm workspaces list
+
+set -e
+
+SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
+REPO="$SCRIPT_DIR/.."
+
+cd "$REPO"
+DIRS=$( jq --raw-output ".workspaces[]" package.json )
```

Comment:
A bit sad to introduce a dependency on `jq` but not the end of the world, especially since it's used only for checking.

Reply:
I guess you're already sad due to https://github.com/SkipLabs/skip/blob/a95d7d8a4fd7a53affe29da584bdd7c03253f43c/sql/test/params/json_args_to_sqlite_params.sh#L11, but also just for testing/checking.